spec/integration/zipgenotypingfiles_spec.rb

```diff
@@ -20,8 +20,13 @@
     subject.perform(phenotype.id, user_phenotype.variation, 'user@example.com')
 
     mail = ActionMailer::Base.deliveries.last
+    
+    it_behaves_like "multipart email"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.